frontend/src/metabase/entities/containers/EntityName.jsx

```diff
@@ -3,11 +3,11 @@ import React from "react";
 
 import EntityObjectLoader from "./EntityObjectLoader";
 
-const EntityName = ({ entityType, entityId, name = "name" }) => (
+const EntityName = ({ entityType, entityId, property = "name" }) => (
```

Comment:
Fix misnamed prop. This doesn't affect the end result since `property` is used to determine if we should refetch an entity based on the required properties. But this change could reduce the number of API calls in theory. Anyway it's always better to fix the typo.